File: Data Structures & Algorithms/climbing-stairs/submission-12.java
class Solution {
    public int climbStairs(int n) {
        if (n<3){
            return n;
        }
        int[] DP=new int[n+1];
        DP[1]=1;
        DP[2]=2;
        for(int i=3;i<DP.length;i++){
            DP[i]=DP[i-1]+DP[i-2];
        }
        return DP[DP.length-1];

    }
}

File: Data Structures & Algorithms/evaluate-reverse-polish-notation/submission-12.py
class Solution:
    def evalRPN(self, tokens: List[str]) -> int:
        stack=[]
        ans=0
        for i in tokens:
            print(stack)
            if i.isnumeric():
                stack.append(int(i))
            elif len(i)>1:
                stack.append(int(i))
            elif i=='+':
                one=stack.pop()
                two=stack.pop()
                stack.append(one+two)
            elif i=='-':
                one=stack.pop()
                two=stack.pop()
                stack.append(two-one)
            elif i=='*':
                one=stack.pop()
                two=stack.pop()
                stack.append(one*two)
            elif i=='/':
                one=stack.pop()
                two=stack.pop()
                stack.append(int(two/one))
        return stack[-1]